web/src/settings_config.ts

```diff
@@ -561,6 +561,7 @@ export const preferences_settings_labels = {
     ),
     fluid_layout_width: $t({defaultMessage: "Use full width on wide screens"}),
     high_contrast_mode: $t({defaultMessage: "High contrast mode"}),
+    realm_enter_sends: new Handlebars.SafeString("<kbd>Enter</kbd>sends when composing a message"),
```

Comment:
`sends` looks too close to the `kbd` element currently. Can we add a space here before `sends`?

Reply:
Done.